docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+services:
+
+  gpt-code-ui:
+    image: ricklamers/gpt-code-ui:0.42.35
```

Comment:
This introduces a new location for hardcoded version number. It should be included in https://github.com/ricklamers/gpt-code-ui/blob/bdd0ecfe41a3693adcd56202159b744513d7ad94/Makefile#L7 or we simply point it to latest